pyproject.toml

```diff
@@ -24,6 +24,7 @@ dependencies = [
   "click >=8.0.0,<9",
   "cryptography >=3.4.4,<37",
   "ecdsa",
+  "frozendict",
```

Comment:
Can you please add lower and upper version bounds?  We don’t have them for all dependencies yet, but we should try to have them for new ones.

Reply:
I checked the generated client code and it comes with a `requirements.txt` file. I figured that they specify more dependencies and also have version bounds. I included the relevant ones.